tests/PKSim.Tests/IntegrationTests/IndividualSpecs.cs

```diff
@@ -100,6 +100,92 @@ public void individual_should_not_have_BSA_parameter()
       }
    }
 
+   public class When_creating_cat : concern_for_Individual
+   {
+      private Individual _beagle;
+
+      public override void GlobalContext()
+      {
+         base.GlobalContext();
+         sut = DomainFactoryForSpecs.CreateStandardIndividual(CoreConstants.Species.CAT);
+         _beagle = DomainFactoryForSpecs.CreateStandardIndividual(CoreConstants.Species.BEAGLE);
+      }
+
+      [Observation]
+      public void individual_species_should_be_cat()
+      {
+         sut.Population.Species.ShouldBeEqualTo(CoreConstants.Species.CAT);
+      }
+
+      [Observation]
+      public void individual_should_not_have_BSA_parameter()
+      {
+         sut.Organism.Parameter(CoreConstants.Parameters.BSA).ShouldBeNull();
+      }
+
+      [Observation]
+      public void all_parameters_with_value_origin_copied_from_beagle_should_have_the_same_value_as_in_beagle()
+      {
+         var allConstCatParameter = sut.Organism.GetAllChildren<Parameter>().Where(p => p.IsConstantParameter()).OrderBy(p=>p.EntityPath()).ToArray();
+         var allConstBeagleParameter = _beagle.Organism.GetAllChildren<Parameter>().Where(p => p.IsConstantParameter()).OrderBy(p => p.EntityPath()).ToArray();
+
+         allConstCatParameter.Length.ShouldBeGreaterThan(0);
+         allConstCatParameter.Length.ShouldBeEqualTo(allConstBeagleParameter.Length);
+
+         for (var i = 0; i < allConstCatParameter.Length; i++)
+         {
+            var catParameter = allConstCatParameter[i];
+            if (catParameter.ValueOrigin.Description != "Copied from Beagle")
+               continue;
+
+            catParameter.Value.ShouldBeEqualTo(allConstBeagleParameter[i].Value, 1e-5);
```

Comment:
Checking that if value origin says "Copied from Beagle", then the value is also the same as in beagle :)
Same for Cattle<=>Minipig comparison